src/main/java/CarRacing/View/RacingGameView.java

```diff
@@ -0,0 +1,61 @@
+package CarRacing.View;
+
+import CarRacing.Domain.Car;
+import CarRacing.Domain.RacingGame;
+import CarRacing.Utils.RandomUtils;
+
+import java.util.List;
+import java.util.Scanner;
+
+
+public class RacingGameView {
+    private static final String START_MESSAGE = "경주할 자동차 이름을 입력하세요.(이름은 쉼표(,) 기준으로 구분)";
+    private static final String TRIAL_MESSAGE = "몇회 시도하시겠습니까?";
+    private static final String RESULT_MESSAGE = "실행결과";
+    private static final String CAR_SITUATION_MESSAGE_FORMAT = "%s : %s%n";
+    private static final String WINNERS_MESSAGE_FORMAT = "%s가 최종 우승했습니다.%n";
+    private static final String LIST_SEPARATOR = ", ";
+    private static final String MOVEMENT = "-";
+    private static final Scanner sc = new Scanner(System.in);
+
+    public static String requestCars(){
+        System.out.println(START_MESSAGE);
+        String result = sc.nextLine();
+        return result;
```

Comment:
```suggestion
        return sc.nextLine();
```
가독성에 문제가 없다면 바로 리턴을 하는 것도 좋습니다 :)